client/src/components/DeleteButton.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const DeleteButton = ({ name, id, isSubscribed, setIsSubscribed }) => {
+  const HOST = 'http://127.0.0.1:5000/';
```

Comment:
this is already declared in common.js, let's just import it from there so we can reuse it

Reply:
I started working on this branch a while before common.js was created. I don't want to go through the headache of a rebase right now so I'll keep it this way for now and make this change sometime in the future.

Reply:
There's no headache of rebase. Alternatively, you can just do `git pull origin main` and that'll merge latest changes into your local branch